src/app/settings/admin/constraint-templates/delete-constraint-template-dialog/component.ts

```diff
@@ -0,0 +1,22 @@
+// Copyright 2020 The Kubermatic Kubernetes Platform contributors.
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//     http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import {Component, Input} from '@angular/core';
+import {ConstraintTemplate} from '@shared/entity/opa';
+
+@Component({
+  selector: 'km-delete-constraint-template-dialog',
+  templateUrl: './template.html',
+  styleUrls: ['./style.scss'],
+})
+export class DeleteConstraintTemplateDialog {
```

Comment:
Do we need specific component for that? Perhaps existing confirmation dialog would be sufficient.

Reply:
Since it needs some styling, probably yes.
Note https://github.com/kubermatic/dashboard/pull/2999#discussion_r562639871

Reply:
you're right. Somehow I didn't saw the warning options

Reply:
done